sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1009,8 +1014,19 @@ case class GpuFromUnixTime(
   override def doColumnar(lhs: GpuColumnVector, rhs: GpuScalar): ColumnVector = {
     // we aren't using rhs as it was already converted in the GpuOverrides while creating the
     // expressions map and passed down here as strfFormat
-    withResource(lhs.getBase.asTimestampSeconds) { tsVector =>
-      tsVector.asStrings(strfFormat)
+    withResource(lhs.getBase.asTimestampSeconds) { secondsVector =>
+      withResource(secondsVector.asTimestampMicroseconds) { tsVector =>
+        if (zoneId.normalized().equals(ZoneId.of("UTC").normalized())) {
```

Comment:
Use ```TimeZoneDB``` utils instead of hard coding UTC here.

Reply:
Done.